src/Experimental/RAGTools/generation.jl

```diff
@@ -139,6 +184,42 @@ function answer!(
     return result
 end
 
+"""
+    answer!(
+        answerer::SimpleAnswerer, index::AbstractManagedIndex, result::AbstractRAGResult;
+        model::AbstractString = PT.MODEL_CHAT, verbose::Bool = true,
+        template::Symbol = :RAGAnswerFromContext,
+        cost_tracker = Threads.Atomic{Float64}(0.0),
+        kwargs...)
+
+Dispatch for `AbstractManagedIndex`.
+"""
+function answer!(
```

Comment:
do we need a separate method? I believe the index isnt used so maybe we just need to allow the new type in the definition?